src/auro-popover.js

```diff
@@ -9,30 +9,89 @@ import { LitElement, html, css } from "lit-element";
 import "focus-visible/dist/focus-visible.min.js";
 import styleCss from "./style-css.js";
 
-// build the component class
-class AuroPopover extends LitElement {
-  // constructor() {
-  //   super();
-  // }
+import Popover from "./tooltip";
```

Comment:
This is the `popover` and `tooltip` thing. It's weird that there is a `tooltip` js file that pulls in popper.js. 

So, naming is hard. what if you just called this popover.js